config/_default/menus/menus.en.toml

```diff
@@ -69,11 +69,17 @@
   parent = "menu-oss"
   weight = 30
 
+[[main]]
+  name = "Open Containers"
+  url = "/open-source/oci/"
+  parent = "menu-oss"
+  weight = 35  
+
```

Comment:
I have mixed feelings about this being in the top-level menu. I see the "sbom" article was added here too, but it feels like they should be together in a section that has more conceptual articles grouped together. Do you plan on having more OCI articles?

The way it looks right now, "Open Container Initiative" is being listed on the left menu together with our open source projects, so it can be misleading. You can see what I mean here: https://deploy-preview-272--ornate-narwhal-088216.netlify.app/open-source/oci/what-is-the-oci/